src/Build.UnitTests/BackEnd/TaskHostConfiguration_Tests.cs

```diff
@@ -419,6 +457,7 @@ public void TestTranslationWithITaskItemArrayInDictionary()
                 "TaskName",
                 @"c:\MyTasks\MyTask.dll",
                 parameters,
+                null,
```

Comment:
It's a good practice to specify the parameter name when passing `null`, `false`, `true` or other literal as an argument.

Reply:
Because when reading the code I have no idea what that parameter is and what type it is.